src/browser.js

```diff
@@ -261,6 +261,12 @@ const updateSidebar = cursor({
   update: Sidebar.update
 });
 
+const closed = (model, result) =>
+  ( result.isOk
+  ? [ model, Effects.none ]
+  : [ model, Effects.perform(Unknown.error(result.error)) ]
```

Comment:
Flow will infer return type of the `closed` function to be `[Model, Tagged<"Errored", void>]` where second item in the tuple will end up being send into `update`. But then update does not actually supports `Action` of that type, which is why type checker fails. On possible fix is to do `.map(NoOp)` as seen on line 692. But honestly our `Unknown.error` function should be updated to make this unnecessary.